templates/navbarProjetIO.html.twig

```diff
@@ -8,12 +8,18 @@
         <ul class="navbar-nav  mr-auto mt-2 mt-lg-0 ">
         </ul>
         <ul class="navbar-nav">
+            {% if user.email is defined %}
```

Comment:
cette partie dans la navbar devrait faire l'objet d'une US séparée !
de plus, pas la peine de récup $this->getUser() dans le controleur pour l'envoyer à la vue si tu n'en fais rien d'autre, car tu peux le recup direct dans la vue avec {{ app.user }}